docs/communication/scenarios.md

```diff
@@ -1,27 +1,181 @@
 # Communication scenarios <!-- omit in toc -->
 
 At Thoth Tech, the Area Leads have identified a number of communication scenarios which can be
-difficult to manage. In each scenario there are different outcomes and courses of action. Below are
-the learnings that came out of these scenarios.
+difficult to manage. In each scenario there are different outcomes and courses of action.
 
 - [Company communication](#company-communication)
 - [Technical difficulties](#technical-difficulties)
 - [Swapping teams](#swapping-teams)
 - [Time zone management](#time-zone-management)
-- [Late on-boarding](#late-on-boarding)
+- [Late onboarding](#late-onboarding)
 - [Conflict management](#conflict-management)
-- [Abuse or harassment](#abuse-or-harassment)
+- [Bullying, abuse, or harassment](#bullying-abuse-or-harassment)
 
 ## Company communication
 
+All Thoth Tech communications to and between company members should be clear and consistent. Most
+communications, including announcements, will be posted in Microsoft Teams. This applies for all
+levels of company communication – the main Thoth Tech channel, product channels, and private team
+channels. As such, these channels and their respective communications should create a flow-down
+effect of communication from Leadership and Managing Directors to all company members.
+
+From such communication, all Thoth Tech members should feel aware of what tasks need to be
+completed, and that they are continually supported and guided. If at any time you are unsure or feel
+that there is insufficient communication regarding tasks, there are several points of contact:
+
```

Comment:
@sarahndyson list items that are not full sentences should not be capitalised, as per the Writing Style Guide